### PR TITLE
Add reddit to social networking websites

### PR DESCRIPTION
**Pull request by skylarmt**

### CODE REVIEW

File: data/technology/social_networking_websites.json [hunk 1]
```diff
@@ -148,6 +148,7 @@
     "Qzone",
     "Raptr",
     "Ravelry",
+    "reddit",
```
Comment (hugovk) [inline data/technology/social_networking_websites.json:151]:
```suggestion
    "Reddit",
```

Comment (skylarmt) [reply data/technology/social_networking_websites.json:151]: Their wordmark and website title both use "reddit", but their brand guidelines use "Reddit".

Comment (hugovk) [reply data/technology/social_networking_websites.json:151]: Yeah, the logo is lowercase, but they use "Reddit" in prose, which is the context Corpora is likely to be used.

(Interesting about the title, Google shows "reddit: the front page of the internet" from desktop and "Reddit" on mobile.)

Anyway, other than the logo, I don't see any "reddit" at, for example, these pages:

https://www.reddithelp.com/en
https://www.redditinc.com/
https://www.redditinc.com/policies
https://redditblog.com/

**Comment by dariusk** [comment]
Ruling in favor of the title-cased "Reddit"! Thanks everyone.

**Pull request merged**